### PR TITLE
Bael 3460

### PR DESCRIPTION
**Pull request by mguarnaccia**

### CODE REVIEW

File: libraries-3/src/main/java/com/baeldung/airline/CommandLine.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+import com.github.rvesse.airline.annotations.Cli;
+import com.github.rvesse.airline.help.Help;
+
+@Cli(name = "baeldung-cli", 
+description = "Baeldung Airline Tutorial",
+defaultCommand = Help.class, 
+commands = { DatabaseSetupCommand.class, LoggingCommand.class, Help.class })
+public class CommandLine {
+
```
Comment (KevinGilmore) [inline libraries-3/src/main/java/com/baeldung/airline/CommandLine.java]: Please use the standard formatter on all the files and correct the indentation. You're using a combination of tabs and spaces.

File: libraries-3/pom.xml [hunk 2]
```diff
@@ -23,6 +23,11 @@
             <artifactId>lombok</artifactId>
             <version>${lombok.version}</version>
         </dependency>
+        <dependency>
+		        <groupId>com.github.rvesse</groupId>
```
Comment (KevinGilmore) [inline libraries-3/pom.xml]: indent only with spaces

File: libraries-3/src/main/java/com/baeldung/airline/CommandLine.java (hunk 1 above)
Comment (mguarnaccia) [reply libraries-3/src/main/java/com/baeldung/airline/CommandLine.java]: Fixed

File: libraries-3/pom.xml (hunk 2 above)
Comment (mguarnaccia) [reply libraries-3/pom.xml]: Fixed. Also moved airline version to a specific parameter

**Comment by sjmillington** [comment]
test this please

**Pull request merged**